CMakeLists.txt

```diff
@@ -76,7 +76,9 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${COMPILER_FLAGS} ${WARNING_FLAGS}")
 
 
 add_subdirectory(Hypodermic)
-add_subdirectory(Hypodermic.Tests)
+if (NOT BUILD_TESTING STREQUAL OFF)
```

Comment:
I think it's good practice to create options for these. I did something similar for my own fork while doing vcpkg packaging:
https://github.com/lerppana/Hypodermic/commit/8b38247f3829aef5dca1bbb4c9d47861a9648de4

Reply:
BUILD_TESTING is a standard option within cmake for this express purpose. Which is also the reason that the conan docks recommend this. Just like it is referenced here https://cmake.org/cmake/help/latest/command/enable_testing.html
I just reuse the option to avoid building any tests unnecessarily.

Reply:
Oh shit you are right, TIL 😅

Reply:
Well this is going to read a bit awkward, but your enthusiastic reaction was one of the few bright spots in a rather annoying day. I am grateful for the oportunity to pass some knowledge to a fellow developer. Thank you, you made someone far away smile today :)